src/components/ContentBlock.tsx

```diff
@@ -24,9 +24,16 @@ const ContentTitle = styled.h3`
   background-color: ${(props) => props.theme.colors.skyBlue};
 `;
 
-const ContentBody = styled.div`
+interface ContentBodyProps {
+  background_color?: string;
+}
+
+const ContentBody = styled.div<ContentBodyProps>`
   background-color: ${(props) =>
-    props.theme.colors.cloudBlue};
+
+    props.background_color !== undefined
```

Comment:
we can check if props.backgroundColor is truthy instead of having to check if its not undefined
`props.backgroundColor ? props.backgroundColor : props.theme.colors.cloudBlue`